geoprocessing/package.json

```diff
@@ -66,8 +68,10 @@
     "@types/geojson": "^7946.0.7",
     "@types/jest": "^26.0.15",
     "@types/lodash": "^4.14.168",
+    "@types/multer": "^1.4.5",
     "@types/node": "^14.14.6",
     "@types/supertest": "^2.0.10",
+    "@types/unzipper": "^0.10.3",
```

Comment:
We should aim at pinning dependencies ([reading](https://docs.renovatebot.com/dependency-pinning/))